system-tests/lib/cre/don/jobs/jobs.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/smartcontractkit/chainlink/system-tests/lib/cre"
 )
 
-func Create(ctx context.Context, offChainClient cldf_offchain.Client, jobSpecs cre.DonJobs) error {
+func Create(ctx context.Context, offChainClient cldf_offchain.Client, donTopology cre.DonTopology, jobSpecs cre.DonJobs) error {
```

Comment:
since we don't use a JD wrapper that contained both JD and a DON, we pass the topology here so that later on we can use node's REST client to accept a jobspec proposal.